src/hooks/useFocus.ts

```diff
@@ -0,0 +1,25 @@
+import { MutableRefObject, useEffect, useRef, useState } from "react";
+
+// Same us https://usehooks.com/useHover/ but for blur and focus
+export default function useFocus(): [MutableRefObject<HTMLElement>, boolean] {
```

Comment:
How come we didn't had this hook? Are we using some other hook for it?

Reply:
We had this: https://style.monday.com/?path=/docs/hooks-usesetfocus--overview
Is there an issue with it? https://style.monday.com/?path=/docs/hooks-usesetfocus--overview